plaid/client.py

```diff
@@ -39,7 +40,8 @@ def __init__(self,
                  suppress_warnings=False,
                  timeout=DEFAULT_TIMEOUT,
                  api_version=None,
-                 client_app=None):
+                 client_app=None,
+                 json_parser=json.loads):
```

Comment:
What do you think about putting this under `_options`? We follow this pattern for other endpoints, and could allow for more flexibility in the future for further overrides (eg: https://github.com/plaid/plaid-python/pull/182)

Reply:
@pbernasconi ping?